src/app/views/query-runner/request/Request.tsx

```diff
@@ -10,6 +10,7 @@ import React, { Component, CSSProperties } from 'react';
 import { injectIntl } from 'react-intl';
 import { connect } from 'react-redux';
 import { bindActionCreators, Dispatch } from 'redux';
+
```

Comment:
extra space ;O

Reply:
@_@

Reply:
oh this was intentional, to seperate library imports and file imports